src/stack-unix/udpv4_socket.ml

```diff
@@ -30,6 +30,7 @@ let get_udpv4_listening_fd {listen_fds;interface} port =
     Lwt.return @@ Hashtbl.find listen_fds (interface,port)
   with Not_found ->
     let fd = Lwt_unix.(socket PF_INET SOCK_DGRAM 0) in
+    Lwt_unix.(setsockopt fd SO_REUSEPORT true);
```

Comment:
Is this to allow multiple Unix processes to bind to the same port, to share responsibility for handling the requests?

Reply:
This is to allow the same unix process to bind to the same port several times, i.e. in DNS:
- we listen on UDP Ipaddr_any, 53 (via a listen callback) -- calls `bind(socket, *, 53)`
- once a UDP frame is received, we take some time to find an answer, and eventually send _from_ (*, 53) to the client -- this means we have to bind as well to (*, 53).

Without the REUSEPORT we can't bind to the same ip, port pair multiple times.